cmd/main.go

```diff
@@ -40,13 +40,13 @@ func init() {
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	config.AddFlags()
 
-	logf.SetLogger(logger.ZapLogger())
 	cf, err = config.NewNSXOperatorConfigFromFile()
 	if err != nil {
-		log.Error(err, "load config file error")
```

Comment:
we still need to keep this error log.

Reply:
For example, the error returned from in func NewNSXOperatorConfigFromFile
func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
	nsxOperatorConfig := NewNSXOpertorConfig()

	cfg := ini.Empty()
	err := ini.ReflectFrom(cfg, nsxOperatorConfig)
	if err != nil {
		return nil, err
	}
	cfg, err = ini.Load(configFilePath)
	if err != nil {
		return nil, err
	}
...
 these error only logged here.  So, we can consider use another logger to log config file info and errors. but this logger need be created before  NewNSXOperatorConfigFromFile and pass it to config file.

Reply:
As discussed, wrap a function and log.error in it.